src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,7 +5,43 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int bodyWidth;
+    private int height;
```

Comment:
Consider adding an access modifier to the 'height' field. Encapsulation is a key principle in object-oriented programming to protect the internal state of an object.